prometheus/prometheus.yml

```diff
@@ -37,6 +37,6 @@ scrape_configs:
       
   - job_name: 'isb-exporter'
     # scrape the iSB metrics once a day
-    scrape_interval: 86400s
+    scrape_interval: 60s
```

Comment:
Getting this down to once a minute -- much easier for debugging problems and shouldn't be a traffic problem.